inst/examples/modules/app.R

```diff
@@ -16,15 +16,20 @@ selectColumn <- function(input, output, session, df) {
 
   values <- metaReactive2({
     req(input$col)
-    metaExpr(`$`(!!df(), !!input$col))
+    metaExpr({
+      ..(df()) %>%
+        dplyr::pull(!!..(input$col))
```

Comment:
I believe this is a result of using `varSelectInput` instead of `selectInput` for `input$col`, but it is a little jarring at first glance to see `!!..(input$col)` in a few instances like this.

Reply:
It is jarring indeed. I don't think it gets better with `selectInput`, I think it would be `!!..(sym(input$col))` in that case :(

Reply:
BTW, I wrote a bit about why this works the way it does in https://github.com/rstudio/shinymeta/blob/joe/experiment/uncouple-at-runtime/vignettes/special-topics.Rmd#L101